test/e2e-tests.sh

```diff
@@ -40,6 +40,59 @@ function dump_extra_cluster_state() {
   kubectl -n knative-build-pipeline logs $(get_app_pod build-pipeline-webhook knative-build-pipeline)
 }
 
+function run_yaml_tests() {
+  echo ">> Starting tests"
+
+  sed -i.bak 's/christiewilson-catfactory/${KO_DOCKER_REPO}/' examples/resources.yaml
```

Comment:
😅 

(btw you know it turns out that `yq` is pretty good at this kinda thing! but i guess we probably dont want to bother vendoring it just for this)